cmd/cloudx/introspect.go

```diff
@@ -0,0 +1,20 @@
+package cloudx
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ory/cli/cmd/cloudx/oauth2"
+	"github.com/ory/x/cmdx"
+)
+
+func NewIntrospectCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "introspect",
```

Comment:
Would the more general verb `describe` also make sense here? I'd imagine that describe would fit more objects than introspect, if we want to follow the `kubectl` model. But I don't know the semantics for sure.